resources/views/inputs/tags.blade.php

```diff
@@ -30,7 +30,7 @@
             ])
         @endunless
 
-        <div class="input-wrapper">
+        <div class="input-wrapper {{ $isDisabled ? 'disabled-input' : '' }}">
```

Comment:
give this a more specific name, as it's a class that disables the taggle tag component elements and not any regular input